### PR TITLE
ADD Packaging doc

### PR DESCRIPTION
**Pull request by dmoranj**

### CODE REVIEW

**Comment by AlvaroVega** [comment]
LGTM

File: docs/installationguide.md [hunk 1]
```diff
@@ -89,3 +89,15 @@ The ones relating specific Ultralight 2.0 bindings are described in the followin
...
+The only package type allowed is RPM. In order to execute the packaging scripts, the RPM Build Tools must be available
+in the system.
+
+From the root folder of the project, create the RPM with the following commands:
+```
+cd rpm
+./create-rpm.sh -v <version-number> -r  <release-number>
+```
+Where `<version-number>` is the version (x.y.z) you want the package to have and `<release-number>` is an increasing
+number dependent un previous installations.
```
Comment (frbattid) [inline docs/installationguide.md]: un --> in ?

File: docs/contribution.md [hunk 2]
```diff
@@ -71,8 +70,9 @@ content of the changelog is also moved to the release description in the Github
 
 ## Releasing
 The process of making a release consists of the following steps:
-1. Create a new task branch changing the development version number in the package.json (with a sufix `-next`), to the
-new target version (without any sufix), and PR into `develop`.
+1. Create and PR into `develop` a new task branch with the following changes:
```
Comment (frbattid) [inline docs/contribution.md]: develop? Does it exist?

**Comment by AlvaroVega** [comment]
LGTM

File: docs/contribution.md [hunk 3]
```diff
@@ -19,42 +19,39 @@ repository, it does not have to be lightweightm2m-iotagent, although we will use
 git remote add lightweightm2m-iotagent https://github.com/telefonicaid/lightweightm2m-iotagent.git
 ```
```
Comment (fgalan) [inline docs/contribution.md]: More changes:

* L14: lightweightm2m-iotagent.git -> iotagent-json.git
* L17: lightweightm2m-iotagent -> iotagent-json
* L18: lightweightm2m-iotagent -> iotagent-json

Comment (AlvaroVega) [reply docs/contribution.md]: Fixed in https://github.com/telefonicaid/iotagent-json/pull/220/commits/d1754b936268238b0b4b992c894cbc677d686e64

**Pull request merged**